src/main/java/de/tum/in/www1/artemis/service/AssessmentService.java

```diff
@@ -262,4 +271,91 @@ public Double calculateTotalScore(List<Feedback> assessments) {
         }
         return totalScore;
     }
+
+    /**
+     * Gets an example submission with the given submissionId and returns the result of the submission.
+     *
+     * @param submissionId the id of the example modeling submission
+     * @return the result of the submission
+     * @throws EntityNotFoundException when no submission can be found for the given id
+     */
+    public Result getExampleAssessment(long submissionId) {
+        Optional<Submission> optionalSubmission = submissionRepository.findExampleSubmissionByIdWithEagerResult(submissionId);
+        Submission submission = optionalSubmission.orElseThrow(() -> new EntityNotFoundException("Example Submission with id \"" + submissionId + "\" does not exist"));
+        return submission.getResult();
+    }
+
+    /**
+     * This function is used for submitting a manual assessment/result. It gets the result that belongs to the given resultId, updates the completion date, sets the assessment type
+     * to MANUAL and sets the assessor attribute. Afterwards, it saves the update result in the database again.
```

Comment:
Maybe I'm missing something, but this does not appear to call `result.setAssessmentType(AssessmentType.MANUAL);`?